src/components/Dropdown/index.tsx

```diff
@@ -1,35 +1,46 @@
 import { Dispatch, SetStateAction } from "react";
 import { useDispatch } from "react-redux";
-import { setIsLoggedIn } from "@/src/store/global";
-
-import { notifyError, notifySuccess } from '../../utils/toast';
-import { DropdownPresentation } from "./Presentation";
+import { setIsLoggedIn, setUserData } from "@/src/store/global";
+import { useLogoutUserMutation } from "@/src/services/logoutApi";
 
 import { DROPDOWN_LINKS } from "./DropdownConstants";
 
-import styles from "./dropdown.module.css";
-import { useLogoutUserMutation } from "@/src/services/logoutApi";
+import { notifyError, notifySuccess } from "../../utils/toast";
+import { DropdownPresentation } from "./Presentation";
 
-export default function Dropdown({ setIsDropdownVisible } : {
-    setIsDropdownVisible: Dispatch<SetStateAction<boolean>>;
+export default function Dropdown({
+  setIsDropdownVisible,
+}: {
+  setIsDropdownVisible: Dispatch<SetStateAction<boolean>>;
 }) {
-    const reduxDispatch = useDispatch();
-    const [ logoutUser ] = useLogoutUserMutation();
+  const reduxDispatch = useDispatch();
+  const [logoutUser] = useLogoutUserMutation();
 
-    const logout = () => {
-        logoutUser()
-          .unwrap()
-          .then(() => {
-            reduxDispatch(setIsLoggedIn({isLoggedIn: false}));
-            notifySuccess('User logged out successfully')
-          })
-          .catch((error) => {
-            const errorMessage = error?.data?.message || 'Something went wrong!';
-            notifyError(errorMessage);
+  const logout = () => {
+    logoutUser()
+      .unwrap()
+      .then(() => {
+        reduxDispatch(setIsLoggedIn({ isLoggedIn: false }));
+        reduxDispatch(
+          setUserData({
+            first_name: null,
+            imageURL: null,
+            roles: null,
           })
-    }
+        );
+        notifySuccess("User logged out successfully");
+      })
+      .catch((error) => {
+        const errorMessage = error?.data?.message || "Something went wrong!";
```

Comment:
If the 'errorMessage' is too long, the toast might break.

Reply:
will keep it Something went wrong as a `errorMessage` toast

Reply:
the Dropdown component being refactored in this PR so will make changes here https://github.com/Real-Dev-Squad/members-site/pull/119